solana-programs/anchor/audius-data/cli/main.ts

```diff
@@ -195,57 +212,85 @@ async function timeCreatePlaylist(args: CreatePlaylistParams) {
   console.log(err);
 }
 
-async function timeUpdatePlaylist(args: UpdatePlaylistParams) {
-  let retries = 5;
-  let err = null;
-  while (retries > 0) {
-    try {
-      const start = Date.now();
-      const tx = await updatePlaylist({
-        program: args.program,
-        playlistPublicKey: args.playlistPublicKey,
-        userStorageAccountPDA: args.userStorageAccountPDA,
-        userAuthorityKeypair: args.userAuthorityKeypair,
-        metadata: args.metadata,
-      });
-      const duration = Date.now() - start;
-      console.log(
-        `Processed ${tx} in ${duration}, user=${options.userStoragePubkey}`
-      );
-      return tx;
-    } catch (e) {
-      err = e;
-    }
-    retries--;
-  }
-  console.log(err);
-}
+// async function timeCreatePlaylist(args: CreateEntityParams) {
```

Comment:
we should prob delete these lines or leave a comment explaining why we're keeping them in if we expect to un-comment them later

Reply:
@isaacsolo lmk if i can do this - i'll add to my branch. thanks for these changes! 🙏

Reply:
commented because they're not done yet! i can finish them up